master/buildbot/config.py

```diff
@@ -21,6 +21,7 @@
 
 from future.utils import iteritems
 from future.utils import itervalues
+from future.utils import string_types
```

Comment:
This [do](http://python-future.org/compatible_idioms.html#basestring)c recommends to use:
```
from past.builtins import basestring
```

which one is better?